2006-08-26  Michael Banck  <mbanck@debian.org>

        * console.c (DEFAULT_ENCODING): Set to UTF-8.

---
 console/console.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/console/console.c
+++ b/console/console.c
@@ -58,7 +58,7 @@ int netfs_maxsymlinks = 16;	/* Arbitrary
 /* Handy source of time.  */
 volatile struct mapped_time_value *console_maptime;
 
-#define DEFAULT_ENCODING "ISO-8859-1"
+#define DEFAULT_ENCODING "UTF-8"
 #define DEFAULT_INTENSITY CONS_ATTR_INTENSITY_NORMAL
 #define DEFAULT_UNDERLINED 0
 #define DEFAULT_BLINKING 0
